tools/QuietBackgroundProcesses/DevHome.QuietBackgroundProcesses.UI/ViewModels/QuietBackgroundProcessesViewModel.cs

```diff
@@ -11,15 +12,24 @@ namespace DevHome.QuietBackgroundProcesses.UI.ViewModels;
 
 public partial class QuietBackgroundProcessesViewModel : ObservableObject
 {
-    private readonly bool _isFeaturePresent;
     private readonly TimeSpan _zero;
+    private readonly TimeSpan _oneSecond;
```

Comment:
NIT:  `readonly` variables can be declared and assigned on the same line.

Reply:
oh nice, you are right.  intellisense doesn't like it, but *shrugs*